freeipa.spec.in

```diff
@@ -1025,6 +1025,15 @@ if [ -e /usr/sbin/ipa_kpasswd ]; then
 # END
 fi
 
+# create users and groups
+# create kdcproxy group and user
+getent group kdcproxy >/dev/null || groupadd -f -r kdcproxy
+getent passwd kdcproxy >/dev/null || useradd -r -g kdcproxy -s /sbin/nologin -d / -c "IPA KDC Proxy User" kdcproxy
```

Comment:
Please specify uid and gid. Otherwise, when used in containers, one image build can end up with different uid/gid than the other, store data in volume with that uid/gid, and the next image can use different values and not be able to read the data.